R/query.R

```diff
@@ -37,25 +37,30 @@
 article_pageviews <- function(project = "en.wikipedia", article = "R (programming language)",
                               platform = "all", user_type = "all",
                               start = "2015100100", end = NULL, reformat = TRUE, ...){
-  
+
+
   article <- gsub(x = article, pattern = " ", replacement = "_", fixed = TRUE)
   if(length(article) > 1){
     return(lapply(article, article_pageviews, project = project, platform = platform,
                   user_type = user_type, start = start, end = end, ...))
   }
+
   # Handle timestamps
+  start <- pageview_timestamps(start)
```

Comment:
What's the purpose of these calls, here? Is it to validate the ideally-now-character-string'd timestamps?